src/os/bluestore/bluestore_tool.cc

```diff
@@ -766,6 +781,23 @@ int main(int argc, char **argv)
       exit(EXIT_FAILURE);
     }
   }
+  else if (action == "show-label-at") {
+    JSONFormatter jf(true);
+    bluestore_bdev_label_t label;
+    int r = BlueStore::read_bdev_label_at_pos(cct.get(), devs[0], offset, &label);
```

Comment:
It would be nice if we verified passed offset, and if it was none of the expected once, we threw a warning.
Something like:
"Suspicious offset: %, expected 0, 1073741824, 10737418240, 107374182400 or 1073741824000".
But action will still be executed.

Reply:
Done